pkg/transport/types/transport.go

```diff
@@ -54,6 +54,9 @@ const (
 
 	// TransportTypeSSE represents the SSE transport.
 	TransportTypeSSE TransportType = "sse"
+
+	// TransportTypeBridge represents the bridge network mode for container networking.
+	TransportTypeBridge TransportType = "bridge"
```

Comment:
What's this piece? I don't think we'll be supporting a bridge transport.

Reply:
Yeah, I'll be removing this. It was added simply because the Inspector server doesn't fit into the sse or stdio category. Just wanted to push what I had up and draft PR it so I can checkout other branches in the meantime.

Reply:
aha! Let's document that a little better. If it really doesn't fit we can re-evaluate.